Bottom: 294ca3f6b98aea6df8aa88e4a8e86adbe30e523b
Top:    85da29bafe028daa3c36cba080c34c4669d4af9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-03 01:29:38 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 1771b93..e3a0d6f 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -1064,9 +1064,6 @@ hidgl_clean_unassigned_stencil (void)
   CHECK_IS_IN_CONTEXT ();
   glPushAttrib (GL_STENCIL_BUFFER_BIT);
   glStencilMask (~assigned_bits);
-//  if (assigned_bits != 0) {
-//    printf ("Doing masked stencil clear :(\n");
-//  }
   glClearStencil (0);
   glClear (GL_STENCIL_BUFFER_BIT);
   glPopAttrib ();
